core-bundle/contao/dca/tl_article.php

```diff
@@ -386,7 +386,7 @@ public function getActiveLayoutSections(DataContainer $dc)
 				// Find all sections with an article module (see #6094)
 				foreach ($arrModules as $arrModule)
 				{
-					if ($arrModule['mod'] == 0 && $arrModule['enable'])
+					if ($arrModule['mod'] == 0 && ($arrModule['enable'] ?? false))
```

Comment:
```suggestion
					if ($arrModule['mod'] == 0 && ($arrModule['enable'] ?? null))
```
AFAIK this is our coding style ☺️

Reply:
Updated :) although I would say it should be false, because we are checking against the boolean value.